### PR TITLE
Modified library to work with GoogleAppEngine

### PR DESCRIPTION
**Pull request by Winghin2517**
This will allow for the current android library to work on GoogleAppEngine - I have made two major change:
1. I have replaced the HTTPClient methods from org.apache with the java.net HTTPUrlConnection methods. 
2. Any java.awt images methods were replaced by the Images API methods of GoogleAppEngine.

### CODE REVIEW

File: src/main/java/se/walkercrou/places/GooglePlaces.java [hunk 1]
```diff
@@ -255,7 +253,7 @@ public GooglePlaces(String apiKey, RequestHandler requestHandler) {
      * @param apiKey that has been registered on the Google Developer Console
      */
     public GooglePlaces(String apiKey) {
-        this(apiKey, new DefaultRequestHandler());
+        this(apiKey, new GAERequestHandler());
     }
```
Comment (Winghin2517) [inline src/main/java/se/walkercrou/places/GooglePlaces.java]: Hello, I was thinking of making the default constructor to be DefaultRequestHandler() however, I have gone and customized your library for GoogleAppEngine which means that I have used classes such as com.google.appengine.api.images.Image and import com.google.appengine.api.images.ImagesServiceFactory to replace your awt.images code - see changes made in the Photo.java class. 
A person who wishes to use this library on GAE can therefore not easy just substitute their requestHandler for the GAERequestHandler and make it work on GAE....Would you be able to pull this request to a new branch on your side for a separate library just for GAE?

**Comment by eman3488** [comment]
Any status on this?

**Comment by windy1** [comment]
@eman3488 There are conflicts in this PR that must be fixed. I would also like to do my own testing on this issue before I pull this but I've been really caught up with school and life lately. Hopefully I will get around to it within the next couple of weeks. I'm going to close this for now but if the conflicts are resolved I will gladly re-open it. Thanks.

**Comment by eman3488** [comment]
Thanks Walker. I might look at what was done originally and take a crack at
it myself branching off the latest code. Good luck with life and school :)

-Eric

**Pull request closed without merging**